src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -47,6 +47,11 @@ public struct GCExtendedGamepadSnapShotDataV100 {
 		public float /* float_t = float */ LeftTrigger;
 		public float /* float_t = float */ RightTrigger;
 
+		[TV (12, 1), Mac (10, 14, 1, onlyOn64: true), iOS (12, 1)]
+		public bool LeftThumbstickButton;
+		[TV (12, 1), Mac (10, 14, 1, onlyOn64: true), iOS (12, 1)]
+		public bool RightThumbstickButton;
```

Comment:
That's a `struct` so it's memory representation must be identical between managed and native. 

That's not the case here. That's an Apple issue (please file rdar) since the fields where added in the **middle** of the structure, i.e. code compiled with the 12.1 headers won't match the runtime memory representation on iOS 12 (some fields will be shifted).

Reply:
Yea that makes total sense. Will try to keep size/alignment concerns in mind next time I update a struct (:

Reply:
https://trello.com/c/7FoGTORD